### PR TITLE
Adding info for conditions for autodiscovery

### PR DESCRIPTION
**Pull request by gizas**
Small section that adds some info for autodiscovery template definition for K8s

### CODE REVIEW

File: docs/en/ingest-management/elastic-agent/configuration/autodiscovery/kubernetes-conditions-autodiscover.asciidoc [hunk 1]
```diff
@@ -1,7 +1,13 @@
 [[conditions-based-autodiscover]]
 = Conditions based autodiscover
 
-You can define autodiscover conditions in each input to allow {agent} to automatically identify Pods and start monitoring them using predefined integrations. Refer to <<elastic-agent-input-configuration>> to get an idea .
+You can define autodiscover conditions in each input to allow {agent} to automatically identify Pods and start monitoring them using predefined integrations. Refer to <<elastic-agent-input-configuration>> to get an idea.
+
+IMPORTANT: Condition definintion is supported both in *{agent} by {fleet}* and in *standalone* scenarios.
```
Comment (kilfoyle) [inline docs/en/ingest-management/elastic-agent/configuration/autodiscovery/kubernetes-conditions-autodiscover.asciidoc:6]:
```suggestion
IMPORTANT: Condition definintion is supported both in *{agent} managed by {fleet}* and in *standalone* scenarios.
```

File: docs/en/ingest-management/elastic-agent/configuration/autodiscovery/kubernetes-conditions-autodiscover.asciidoc [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
+
+Read more information for variables and conditions in input configurations in <<dynamic-input-configuration>>
+You can find available variables of autodiscovery in <<kubernetes-provider>>
```
Comment (kilfoyle) [inline docs/en/ingest-management/elastic-agent/configuration/autodiscovery/kubernetes-conditions-autodiscover.asciidoc:9]:
```suggestion
For more information about variables and conditions in input configurations, refer to <<dynamic-input-configuration>>.
You can find available variables of autodiscovery in <<kubernetes-provider>>.
```
Just rephrasing a little bit to match what we have in some other docs.

**Review by constanca-m — APPROVED** [review]

**Review by kilfoyle — DISMISSED** [review]
LGTM! 🚢

File: docs/en/ingest-management/elastic-agent/configuration/autodiscovery/kubernetes-conditions-autodiscover.asciidoc [hunk 3]
```diff
@@ -1,7 +1,13 @@
 [[conditions-based-autodiscover]]
 = Conditions based autodiscover
 
-You can define autodiscover conditions in each input to allow {agent} to automatically identify Pods and start monitoring them using predefined integrations. Refer to <<elastic-agent-input-configuration>> to get an idea .
+You can define autodiscover conditions in each input to allow {agent} to automatically identify Pods and start monitoring them using predefined integrations. Refer to <<elastic-agent-input-configuration>> to get an idea.
+
+IMPORTANT: Condition definintion is supported both in *{agent} managed by {fleet}* and in *standalone* scenarios.
```
Comment (gsantoro) [inline docs/en/ingest-management/elastic-agent/configuration/autodiscovery/kubernetes-conditions-autodiscover.asciidoc:6]:
```suggestion
IMPORTANT: Condition definition is supported both in *{agent} managed by {fleet}* and in *standalone* scenarios.
```

**Review by gsantoro — CHANGES_REQUESTED** [review]
minor type

**Comment by gizas** [comment]
@kilfoyle same issue I can not merge. So if you dont have any objection please ...

**Review by gsantoro — APPROVED** [review]

**Pull request merged**